src/Components/pollutionDataPage/PollutionData.module.css

```diff
@@ -0,0 +1,135 @@
+.contianer {
+  padding-top: 10px;
+  height: 100%;
+}
+
```

Comment:
- [ ] Your app works great! However, I think the styling of the list on the Homepage can be improved to conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), by ensuring that the cards of the list follow the alternating dual-color pattern. Please bear in mind that one of the objectives of this capstone is that our design matches the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Let's therefore ensure that the cards of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (or slightly alternating color shade) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/219722492-3cde13b8-14d5-4f0b-8b7b-f59b2a44cca3.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png">

Reply:
Done 👌